### PR TITLE
Read from environment

### PR DESCRIPTION
**Pull request by mhogerheijde**
Read both PORT and KEY from environment

And put user-agent in _outgoing_ requests.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -98,8 +98,8 @@ var config = configure()
 
 var server = require('webserver').create();
 var urlPrefix = config['url'];
-var port = config['port'] || system.env.PORT || 8082;
-var key = config['key'];
+var port = system.env.PORT || config['port'] || 8082;
+var key =  system.env.KEY || config['key'];
```
Comment (SAGV) [inline server.js:105]: So now we have 5 official ways to set port? :-)

File: server.js [hunk 2]
```diff
@@ -98,8 +98,8 @@ var config = configure()
...
 var userAgent = config['user_agent'];
```
Comment (SAGV) [inline server.js:106]: Should we also add ability to get user_agent and host from env?

File: server.js (hunk 1 above)
Comment (mhogerheijde) [inline server.js:105]: Eehm. I guess so, we have
1. Hardcoded
2. config.json
3. /etc/prerender.json
4. config-local.json
5. environment

So I guess for current Heroku setup:
1. We could just "fail" if no value was found,
2. should be left there I guess, it is handy for having a place to put settings
3. could be cut out for heroku, but handy for deployment on non-heroku setups 
4. is really handy for local development, 
5. is needed for Heroku...

File: server.js (hunk 2 above)
Comment (mhogerheijde) [inline server.js:106]: I don't think that that is something we would change often, but do want to change outside of source-code. For instance, we could do something like "GoAbout/Prerenderer v1.0" and move to "GoAbout/Prerenderer v1.1"... But I don't think we'd want to set different values across Heroku machines?

The KEY thing is something I don't want to end up in a public place.

**Comment by mhogerheijde** [comment]
@SAGV added extra fix ;)

**Pull request merged**